specs/ReleaseChannelPreferenceSet.md

```diff
@@ -0,0 +1,207 @@
+that the WebView2 loader should search for in order of least to most stable:
+Canary -> Dev -> Beta -> Stable WebView2 Runtime. For example, you can specify {Beta, Stable}
+to tell the WebView2 loader to try running the app with Beta channel if it exists on the machine,
+and otherwise fall back on the stable WebView2 Runtime. With `ReleaseChannelPreferenceSet` you can:
+- Run automated tests against different channels on a machine with multiple channels installed
+- Target a particular channel when flighting
+- Apply different channels to different flighting rings
+- Fall back on a more stable preview channel
+
+# Examples
+## ReleaseChannelPreferenceSet
+The release channel preference set is specified on the CoreWebView2EnvironmentOptions prior to
+creating the WebView2 environment.
+```c#
+CoreWebView2Environment _webViewEnvironment;
+async System.Threading.Tasks.Task CreateEnvironmentAsync() {
+    List<CoreWebView2ReleaseChannel> channels = new List<CoreWebView2ReleaseChannel>
+    {
+        CoreWebView2ReleaseChannel.Beta
+        CoreWebView2ReleaseChannel.Stable,
+    };
+    CoreWebView2EnvironmentOptions customOptions = new CoreWebView2EnvironmentOptions
+    {
+        ReleaseChannelPreferenceSet = channels
+    };
+    // If the loader is unable to find a valid installation from the release
+    // channel preference set, environment creation will fail.
+    _webViewEnvironment = await CoreWebView2Environment.CreateAsync(
+            options: customOptions);
+    if (_webViewEnvironment != null)
+    {
+        CheckChannel(_webviewEnvironment.BrowserVersionString());
+    }
+}
+```
+```cpp
+void AppWindow::InitializeWebViewEnvironment()
+{
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    std::vector<COREWEBVIEW2_RELEASE_CHANNEL> channels =
+    {
+        COREWEBVIEW2_RELEASE_CHANNEL_BETA,
+        COREWEBVIEW2_RELEASE_CHANNEL_STABLE
+    };
+    CHECK_FAILURE(options->SetReleaseChannelPreferenceSet(
+        channels.size(), channels.data()));
+    // If the loader is unable to find a valid installation from the release
+    // channel preference set, environment creation will fail with
+    // HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND).
+    HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
+        nullptr, m_userDataFolder.c_str(), options.Get(),
+        Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+            this, &AppWindow::OnCreateEnvironmentCompleted)
+            .Get());
+}
+void AppWindow::OnCreateEnvironmentCompleted(
+    HRESULT result, ICoreWebView2Environment* environment)
+{
+    if (environment)
+    {
+        wil::unique_cotaskmem_string version;
+        CHECK_FAILURE(environment->get_BrowserVersionString(&version));
+        CheckChannel(version.get());
+    }
+}
+```
+# API Details
+```
+typedef enum COREWEBVIEW2_RELEASE_CHANNEL {
+  /// The stable WebView2 Runtime that is released every 4 weeks.
```

Comment:
It would be nice to either say in the docs or have the docs link to a doc that provides more info about each of the channels. Its good to say how often they're released like you do but would be better to also say their relative stability. Enough info for end dev to read this and decide which enum value to use for retail / flighting / automated testing and so on